test/fw/ptl/utils/pbs_procutils.py

```diff
@@ -269,21 +282,55 @@ def set_frequency(self, value=60):
         self.logger.debug('procmonitor: set frequency to ' + str(value))
         self.frequency = value
 
+    def get_system_stats(self, nw_protocols=['TCP']):
```

Comment:
Using mutable objects as defaults is not the greatest, as if you mutate the object it will not reset.
Use a sentinel value for the default

Reply:
@vstumpf modified. Is it ok?